src/interfaces.d.ts

```diff
@@ -409,20 +409,21 @@ export interface WidgetMetaConstructor<T extends WidgetMetaBase> {
 	new (properties: WidgetMetaProperties): T;
 }
 
+export interface NodeHandler extends Evented {
```

Comment:
we tend to have doc on exported interfaces (we also tend to suffix with `Interface`)